README.md

```diff
@@ -17,6 +17,12 @@ The scripts are written for Bash and supported on macOS and Linux. See [Document
 ./build.sh
 ```
 
+Once the build is successful, the built SDK tarball is placed at:
+
+    artifacts/${ARCHITECTURE}/Release/dotnet-sdk-${SDK_VERSION}-${RUNTIME_ID}.tar.gz
+
+Where `${ARCHITECTURE}` is your platform architecture (probably `x64`), `${SDK_VERSION}` is the SDK version you are building and `${RUNTIME_ID}` is your OS name and architecture (something like `debian.9-x64` or `fedora.33-x64`). For example, building a 3.1.105 SDK on an x64 (aka x86\_64) platform running Fedora 32 will produce `artifacts/x64/Release/dotnet-sdk-3.1.105-fedora.32-x64.tar.gz`.
```

Comment:
Seems easier to read to me with structure:

```markdown
* `${ARCHITECTURE}` is your platform architecture (probably `x64`)
* `${SDK_VERSION}` is the SDK version you are building
* `${RUNTIME_ID}` is your OS name and architecture (something like `debian.9-x64` or `fedora.33-x64`)

For example, building a 3.1.105 SDK on an x64 (aka x86\_64) platform running Fedora 32 will produce `artifacts/x64/Release/dotnet-sdk-3.1.105-fedora.32-x64.tar.gz`.
```
> Once the build is successful, the built SDK tarball is placed at:
> 
>     artifacts/${ARCHITECTURE}/Release/dotnet-sdk-${SDK_VERSION}-${RUNTIME_ID}.tar.gz
> 
> * `${ARCHITECTURE}` is your platform architecture (probably `x64`)
> * `${SDK_VERSION}` is the SDK version you are building
> * `${RUNTIME_ID}` is your OS name and architecture (something like `debian.9-x64` or `fedora.33-x64`)
> 
> For example, building a 3.1.105 SDK on an x64 (aka x86\_64) platform running Fedora 32 will produce `artifacts/x64/Release/dotnet-sdk-3.1.105-fedora.32-x64.tar.gz`.

Reply:
Thanks for the review! Fixed now.